en/docs/apis/approvals-rest-api.md

```diff
@@ -4,24 +4,25 @@ template: templates/swagger.html
 
 # Workflow Approval API Definition - v1 
 
-!!! note 
-    Follow the instructions given below to try out the REST APIs with your local instance of WSO2 Identity Server. 
+??? note "Click For Instructions"
+    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server. 
```

Comment:
Why has this been changed from "Follow the instructions given below"? IMO "Do the following" sounds incomplete.